CHANGELOG.md

```diff
@@ -16,6 +16,12 @@ Allowed headings:
 
 ## [Unreleased]
 
+## [4.3.0] - 2022-11-21
```

Comment:
I don't think we need to add the version/date to the changelog a this stage. In theory there might be a few changes per release, so I _think_ the release process adds this bit for us when we actually do the release with the date at that time.